staging/istio/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: istio
-version: 1.8.1
+version: 1.8.2
```

Comment:
It would be nice to come up with a different versioning schema in the future. The current approach may be confusing as sometimes the chart version matches the app version sometimes it does not.

Reply:
yeah, I agree. I wished if we could have something like 1.8.1-1. But, I remember that lint doesn't like it

Reply:
Even better is to not even try to make the chart version match the app version. They're not related, nor should they be.